ocs/exceptions.py

```diff
@@ -4,3 +4,7 @@ class CommandFailed(Exception):
 
 class UnsupportedOSType(Exception):
     pass
+
+
+class CommandFailedException(Exception):
```

Comment:
we have CommandFailed Exception in the same file, probably this was overlooked before during the merge, we can use the original one and delete the extra in install.py

Reply:
right,

Fixing now.
So now I just need to:
git commit (no --amend)
and push new commit?

Reply:
@RazTamir yeah you can just push a new commit and upon merging to master if you choose to squash your commits into one github will let you do that